### DISCUSSION TITLE
'hexo' is not an internal or external command, nor is it a executable program or batch file."

### DISCUSSION THREAD

**Discussion by JovanYang** [hexojs/hexo#5539] [Question / Answer] [answered]
### Check List

- [X] I have already read [Docs page](https://hexo.io/docs/) & [Troubleshooting page](https://hexo.io/docs/troubleshooting).
- [X] I have already searched existing issues and they are not help to me.
- [X] I examined error or warning messages and it's difficult to solve.
- [X] I am using the [latest](https://github.com/hexojs/hexo/releases) version of Hexo. (run `hexo version` to check)
- [X] My Node.js is matched [the required version](https://hexo.io/docs/#Required-Node-js-version).

### Expected behavior

show the hexo vesion

### Actual behavior

'hexo' is not an internal or external command, nor is it a executable program or batch file."

### How to reproduce?

write hexo -v command in command envirment.

### Is the problem still there under `Safe mode`?

YES

**Comment by D-Sketon** [comment]
It's a malware attack. Don't click any untrustworthy links.

**Comment by uiolee** [comment, marked as answer]
install hexo-cli globally

```bash
npm i hexo-cli
```

OR

use `npx hexo` instead
